wordpress.org/public_html/wp-content/themes/pub/wporg-support/sass/blocks/_group.scss

```diff
@@ -0,0 +1,14 @@
+.wp-block-group {
+
+	&.is-layout-grid.is-style-cards-grid {
+		grid-template-columns: repeat(auto-fill, minmax(min(32.3%, 100%), 1fr));
```

Comment:
Should be able to use block markup for this actually, see https://github.com/WordPress/wporg-developer/blob/trunk/source/wp-content/themes/wporg-developer-2023/patterns/front-page-content.php#L17